en/components/general/getting_started.md

```diff
@@ -32,68 +37,52 @@ To get started, you must install both NodeJS and Visual Studio Code on your mach
 </div>
 <div class="divider--half"></div>
 
-## Installing Ignite UI for Angular
+### Installing Ignite UI for Angular
 
-Now that we have everything we need, we can start creating/updating our application!
+The Ignite UI for Angular can be installed either wth the Angular CLI or with the Ignite UI CLI.
 
-### Create an application by using the Ignite UI CLI
+#### Quick Start with the Angular CLI
 
-Let's start by opening a preferred terminal and installing the [`Ignite UI CLI`](https://github.com/IgniteUI/igniteui-cli):
+The Angular CLI provides support for external libraries to your project through the ng add command, which installs library's npm packages to your workspace and configures the project in the current working directory to use that library.
+
+To create an Angular application with the Angular CLI open your preferred terminal application and type the following command:
 
 ```cmd
-npm install -g igniteui-cli 
+ng new <project name> --style=scss
 ```
+You can specify the file extension or preprocessor to use for your application's style files by the --style option. We recommend using Scss since our components' styles are based on the [Ignite UI for Angular theming library](../themes.md). When you install Ignite UI package your application will be configured to use the default styling theme which can be then easily customized either for all or for specific component instances.
 
-#### Using guided experience
-
-The shortest and easiest way to bootstrap an application is to use the Ignite UI CLI [`guided experience`](https://github.com/IgniteUI/igniteui-cli/wiki/step-by-step), which builds a configured app that the developer can get up and running with the ease of a single command. 
+Thereafter you can install the Ignite UI for Angular package, along with all of its dependencies, font imports and styles references to your project by the following command: 
 
 ```cmd
-ig
+ng add igniteui-angular
 ```
 
-<div style="display:inline-block;">
-    <a style="background: url(../../images/general/buildCLIapp.gif); display:flex; justify-content:center; min-width:540px; min-height:315px;"
-       href="https://youtu.be/QK_NsdtdA70" target="_blank">
-        <img src="../../images/general/play.svg" style="vertical-align: middle;" />
-    </a>
-    <p style="text-align:center;">Building Your First Ignite UI CLI App</p>
-</div>
+During the installation process you will be asked if you would like to enable polyfills for IE, Edge and Safari. This will install the web-animations-js package in your project, which is required if you use AnimationBuilder in your application and one of these browsers. You can also choose to add CSS library to reset HTML element styles across browsers and the minireset.css will be installed in your application.
```

Comment:
```suggestion
During the installation process you will be asked if you would like to enable polyfills for IE, Edge and Safari. This will install the `web-animations-js` package in your project, which is required if you use `AnimationBuilder` in your application and one of these browsers. You can also choose to add CSS library to reset HTML element styles across browsers and the `minireset.css` will be installed in your application.
```